geocoder_tester/world/italy/test_synonyms.csv

```diff
@@ -0,0 +1,6 @@
+query,lon,lat,limit,lang,expected_name,expected_housenumber,expected_street,expected_city
+C.po san polo venezia,,,4,it,,,Campo San Polo,Venezia
+L.go Guglielmo Henzen Roma,,,4,it,,,Largo Guglielmo Henzen,Roma
+Via del fontile Anagnino Roma,,,4,it,,,Via del Fontanile Anagnino,Roma
```

Comment:
isn't it more a typo than a synomyme ? (I'm not an italian expert :stuck_out_tongue_winking_eye: )

Reply:
found it https://github.com/openvenues/libpostal/blob/01afbf80efd9baf2adbe62c97901b4fd98f8854b/resources/dictionaries/it/synonyms.txt#L41